src/functionalTest/resources/application.yaml

```diff
@@ -14,6 +14,14 @@ azure:
       write-access-key-name: ${PAYMENTS_QUEUE_WRITE_ACCESS_KEY_NAME}
       queue-name: ${PAYMENTS_QUEUE_NAME}
 
+bulk-scan-procesor:
+  api:
+    retries: 10
+    wait-time-in-ms: 1000
+
+async:
+  threadpool-size:5
+
```

Comment:
all of them are related, wee can associate them?

Reply:
associated with bulk-scan-processor